modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -716,3 +716,8 @@ This also corresponds to the median of the query execution time.
 m| MAX
 |===
 
+[NOTE]
```

Comment:
Maybe we should would put that to the end of the section "Metrics granularity" because it is more relevant to that?